src/Controllers/ModuleSuiteLocator.php

```diff
@@ -190,4 +196,30 @@ protected function loadSuiteConfiguration($suite, Module $module)
             'settings' => $resolvedConfig,
         ];
     }
+
+    private function setI18nLocale(array $config): void
+    {
+        $path = $config['type']['settings']['paths'][0] ?? '';
+        $packagistName = '';
+        if (preg_match('#^vendor/([^/]+/[^/]+)/.+#', $path, $matches)) {
+            // Running behat tests of a module in the vendor folder
+            $packagistName = $matches[1];
+        } else {
+            // Running behat tests of a module or project in the root folder
+            $file = BASE_PATH . '/composer.json';
+            if (file_exists($file)) {
+                $json = json_decode(file_get_contents($file), true);
+                $packagistName = $json['name'] ?? '';
+            }
+        }
+        $metaData = MetaData::getMetaDataByPackagistName($packagistName);
+        $isSupportedModule = !empty($metaData);
+        if ($isSupportedModule) {
+            // Anything that is in silverstripe/supported-module has behat tests in en_US
+            // Update the default_locale config in case in case any config at the project level or any
+            // installed optional module has set it to a non-en_US locale
+            i18n::config()->set('default_locale', 'en_US');
+            i18n::set_locale('en_US');
```

Comment:
Is the kernel booted by this stage? Will this work for the in-browser test instances which boot separate kernels from the one running the CLI instance?

Reply:
I think so though honestly I don't know. On my local which uses a headless chrome inside the docker container it seems to works fine i.e. broken behat tests linked to in the description now pass, and it also works fine on CI which is a similar sort of setup.